README.md

```diff
@@ -242,40 +162,18 @@ Feel free to check the [issues page](../../issues/).
 
 ## ⭐️ Show your support <a name="support"></a>
 
-> Write a message to encourage readers to support your project
-
-If you like this project...
+> Give a ⭐️ if you like this project!
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- ACKNOWLEDGEMENTS -->
 
 ## 🙏 Acknowledgments <a name="acknowledgements"></a>
 
```

Comment:
- Please, let's give appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/) to make our readme more professional.

Reply:
### TESTS

- Your tests are not passing. Please, let's try fixing this issue.


![Screenshot from 2022-12-02 20-44-09](https://user-images.githubusercontent.com/59397159/205374591-60f37df9-3b09-47b4-8b1e-adcc98b93f61.png)